tests/draft7/optional/format/iri.json

```diff
@@ -25,9 +25,14 @@
             },
             {
                 "description": "a valid IRI based on IPv6",
-                "data": "http://2001:0db8:85a3:0000:0000:8a2e:0370:7334",
+                "data": "http://[2001:0db8:85a3:0000:0000:8a2e:0370]:7334",
```

Comment:
This is not a valid IRI as `2001:0db8:85a3:0000:0000:8a2e:0370` is not a valid IPv6 (8 segments required).
`http://[2001:0db8:85a3:0000:0000:8a2e:0370:7334]` this is correct IRI (:7334 is not a port).

Reply:
Good catch! Thanks @vearutop